pkg/visor/visorconfig/v1.go

```diff
@@ -72,7 +72,7 @@ type LogStore struct {
 
 // Routing configures routing.
 type Routing struct {
-	SetupNodes         []cipher.PubKey `json:"setup_nodes,omitempty"`
+	RouteSetupNodes    []cipher.PubKey `json:"route_setup_nodes,omitempty"`
```

Comment:
This would necessitate a change in the config file, right? I generally agree that `route setup node` is more explicit and good, but have we prepared all needed changes to migrate existing configs?

Reply:
The config is expected to be updated on installation or updates. This happens automatically by default in the packages and installers, as far as I'm aware.

Or else the user would just need to regenerate their config.